src/main/java/jp/co/future/uroborosql/context/SqlContextFactory.java

```diff
@@ -10,44 +10,47 @@
 import java.util.Map;
 import java.util.function.Consumer;
 
+import jp.co.future.uroborosql.config.SqlConfigAware;
 import jp.co.future.uroborosql.filter.SqlFilterManager;
 import jp.co.future.uroborosql.parameter.Parameter;
 import jp.co.future.uroborosql.parameter.mapper.BindParameterMapper;
 
 /**
- * SQLコンテキストファクトリインタフェース
+ * SQLコンテキストファクトリインタフェース.
  *
  * @author H.Sugimoto
  *
  */
-public interface SqlContextFactory {
+public interface SqlContextFactory extends SqlConfigAware {
 	/** BEAN名 */
 	String FACTORY_BEAN_NAME = "sqlContextFactory";
 
 	/**
-	 * 初期化処理
+	 * 初期化処理.
 	 */
 	void initialize();
 
 	/**
-	 * SQLコンテキスト生成メソッド
+	 * SQLコンテキスト生成メソッド.
 	 *
 	 * @return SQLコンテキスト
 	 */
 	SqlContext createSqlContext();
 
 	/**
-	 * SQLフィルタ管理クラスを取得する
+	 * 定数パラメータプレフィックスを取得.
 	 *
 	 * @return SQLフィルタ管理クラス
 	 */
+	@Deprecated
```

Comment:
@HidekiSugimoto189 

If you add `@Deprecated`, please provide an alternative in Javadoc with `@deprecated`.

Reply:
@shout-star 
that's right.
I added.